index.html

```diff
@@ -0,0 +1,47 @@
+
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="vendor/normalize.css" type="text/css" rel="stylesheet">
+    <link href="styles/base.css" type="text/css" rel="stylesheet">
+    <link href="styles/layout.css" type="text/css" rel="stylesheet">
+    <link href="styles/modules.css" type="text/css" rel="stylesheet">
+    <link href="styles/icons/icons.css" type="text/css" rel="stylesheet">
+    <title>Peter's Portfolio</title>
+  </head>
+  <body>
+<header>
```

Comment:
Your header indentation is off. It should line up with the section tag below it.